tutorials/intro/plot_projectors_intro.py

```diff
@@ -0,0 +1,392 @@
+# at each sampling time), where :math:`N` is usually in the tens or hundreds
+# (depending on how many sensors your EEG/MEG system has). Fortunately, because
+# projection is a matrix operation, it can be done very quickly even on signals
+# with hundreds of dimensions and tens of thousands of time points.
+#
+#
+# .. _ssp-tutorial:
+#
+# Signal-space projection (SSP)
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+#
+# We mentioned above that the projection matrix will vary depending on what
+# kind of noise you are trying to project away. Signal-space projection (SSP)
+# [1]_ is a way of estimating what that projection matrix should be, by
+# comparing measurements with and without the signal of interest. For example,
+# you can take additional "empty room" measurements that record activity at the
+# sensors when no subject is present. By looking at the spatial pattern of
+# activity across MEG sensors in an empty room measurement, you can create one
+# or more :math:`N`-dimensional vector(s) giving the "direction(s)" of
+# environmental noise in sensor space (analogous to the vector for "effect of
+# the trigger" in our example above).
+#
+# Once you know those vectors, you can create a hyperplane that is orthogonal
+# to them, and construct a projection matrix to project your experimental
+# recordings onto that hyperplane. In that way, the component of your
+# measurements associated with environmental noise can be removed. Again, it
+# should be clear that the projection reduces the dimensionality of your data —
+# you'll still have the same number of sensor signals, but they won't all be
+# *linearly independent* — but typically there are tens or hundreds of sensors
+# and the noise subspace that you are eliminating has only 3-5 dimensions, so
+# the loss of degrees of freedom is usually not problematic.
+#
+#
+# Projectors in MNE-Python
+# ^^^^^^^^^^^^^^^^^^^^^^^^
+#
+# In our example data, :ref:`SSP <ssp-tutorial>` has already been performed
+# using empty room recordings, but the :term:`projectors <projector>` are
+# stored alongside the raw data and have not been *applied* yet (or,
+# synonymously, whether the projectors are *active* or not). Here we'll load
+# the :ref:`sample data <sample-dataset>` and crop it to 60 seconds; you can
+# see the projectors in the output of :func:`~mne.io.read_raw_fif` below:
+
+sample_data_folder = mne.datasets.sample.data_path()
+sample_data_raw_file = os.path.join(sample_data_folder, 'MEG', 'sample',
+                                    'sample_audvis_raw.fif')
+raw = mne.io.read_raw_fif(sample_data_raw_file)
+raw.crop(tmax=60).load_data()
+
+###############################################################################
+# In MNE-Python, the environmental noise vectors are computed using `principal
+# component analysis`_, usually abbreviated "PCA", which is why the SSP
+# projectors usually have names like "PCA-v1". The projectors are stored in the
```

Comment:
I would also mention the acronym SVD as it is common to say that SVD was used for projection
computation.